pkg/common/common.go

```diff
@@ -213,6 +213,8 @@ func StripDefaultPorts(fromUrl string) (string, error) {
 		URL: u,
 	}
 	request.SanitizeHostForHeader(&r)
-	r.URL.Host = r.Host
+	if r.Host != "" {
```

Comment:
I would add unit tests to some scenarios (IBM, Noobaa, etc) here https://github.com/openshift/oadp-operator/blob/master/pkg/common/common_test.go to avoid regressions

Reply:
Tested, and this fixes my use case :smile: 

Only concern I have with follow up PR is that we need this in 1.2 branch right? then will be 3 cherry-picks

Reply:
3 cherry-picks to oadp-1.2 and oadp 1.1 and what's the other one?

But yeah follow up pr is fine.

Reply:
- [ ] add unit tests to some scenarios (IBM, Noobaa, etc) here https://github.com/openshift/oadp-operator/blob/master/pkg/common/common_test.go to avoid regressions

Reply:
oh no, I think no need to 1.1, only for 1.2

it would be form this PR, the previous (https://github.com/openshift/oadp-operator/pull/1169) and the follow up (3, but for the same branch)